contracts/KashiPair.sol

```diff
@@ -711,11 +711,13 @@ contract KashiPair is ERC20, BoringOwnable, IMasterContract {
         } else {
             // Swap using a swapper freely chosen by the caller
             // Open (flash) liquidation: get proceeds first and provide the borrow after
-            bentoBox.transfer(collateral, address(this), swapper != ISwapper(0) ? address(swapper) : to, allCollateralShare);
+            bentoBox.transfer(collateral, address(this), swapper == ISwapper(0) ? to : address(swapper), allCollateralShare);
             if (swapper != ISwapper(0)) {
                 swapper.swap(collateral, asset, msg.sender, allBorrowAmount, allCollateralShare);
             }
 
+            uint256 share = bentoBox.toShare(asset, allBorrowAmount, true);
```

Comment:
@boringcrypto @Clearwood 
Is this correct? Basically we 'lost' assets on borrow and we gather (some) back here. As I see it,  that should be reflected to `totalAsset.elastic`.